### DISCUSSION TITLE
v3 delay in scroll

### DISCUSSION THREAD

**Discussion by againwiththeusername** [noah-nuebling/mac-mouse-fix#984] [Questions]
Hello all - just upgraded to V3. (V2 worked perfectly). With V3 there is a delay of about a second or so between scrolling the mouse scroll button and the actual on-screen scroll. Any and all suggestions welcome ... thanks :)

**Comment by wjwyatt** [comment]
Same issue here.  Any update?

**Comment by againwiththeusername** [comment]
Sorry - no updates, I ended up changing my computer (for a variety of reasons).
